lms/djangoapps/discussion_api/tests/utils.py

```diff
@@ -61,6 +61,55 @@ def register_get_thread_response(self, thread):
             status=200
         )
 
+    def register_post_comment_response(self, response_overrides, thread_id=None, parent_id=None):
```

Comment:
I noticed that you used `response_overrides` in some of the response helpers and `response` in the minimal creation helpers. Any reason for the distinction?